pyproject.toml

```diff
@@ -1,13 +1,16 @@
-[tool.poetry]
```

Comment:
[Noticed](https://python-poetry.org/docs/pyproject/#name-1) that `tool.poetry#name` is deprecated in favor of `project#name`.